server/src/main/resources/application.yml

```diff
@@ -33,11 +33,11 @@ extender:
         update-interval: 900000
 
 spring:
-    http:
+    servlet:
```

Comment:
I never saw this in the migration guides :/

I got upload issues when trying to build the rive extension, but not in my other smaller tests:(